doc/api/dns.md

```diff
@@ -1010,6 +1010,13 @@ An error will be thrown if an invalid address is provided.
 The `dnsPromises.setServers()` method must not be called while a DNS query is in
 progress.
 
+Note that this method works much like
+[resolve.conf](http://man7.org/linux/man-pages/man5/resolv.conf.5.html).
+That is, if attempting to resolve with the first server provided results in a
+`NOTFOUND` error, the `resolve` method will *not* attempt to resolve with
```

Comment:
Nit: `` `resolve` `` -> `` `resolve()` ``?
(see [STYLE_GUIDE.md](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md) about references to methods format)

Reply:
Sorry, did not see that.